Gemfile.lock

```diff
@@ -1,55 +1,55 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.3)
+      actionview (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.3)
+      activesupport (= 6.0.3.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activesupport (6.0.3.2)
+    activemodel (6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activerecord (6.0.3.3)
+      activemodel (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activesupport (6.0.3.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     builder (3.2.4)
-    concurrent-ruby (1.1.6)
+    concurrent-ruby (1.1.7)
     crass (1.0.6)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.3)
+    i18n (1.8.5)
       concurrent-ruby (~> 1.0)
-    json (2.3.0)
-    jwt (2.2.1)
-    jwt_signed_request (2.5.4)
+    json (2.3.1)
+    jwt (2.2.2)
+    jwt_signed_request (2.6.0)
```

Comment:
### [jwt_signed_request](https://github.com/envato/jwt_signed_request)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.5.4 → 2.6.0

[_[change-log](https://github.com/envato/jwt_signed_request/blob/master/CHANGELOG.md), [source-code](https://github.com/envato/jwt_signed_request)_]





<details>
<summary>Commits</summary>

A change of **33** commits. See the full changes on [the compare page](https://github.com/envato/jwt_signed_request/compare/v2.5.4...v2.6.0).

These are the first 10 commits:
- (6601a2c) [Avoid nesting specs in module](https://github.com/envato/jwt_signed_request/commit/6601a2cc0f2ba3143c8ccd97fa1258a37b543019)
- (acd7544) [Formatting](https://github.com/envato/jwt_signed_request/commit/acd754404a58a4124fe617ba101f154d433fcfa3)
- (d08632f) [Move subject etc to top](https://github.com/envato/jwt_signed_request/commit/d08632fd823584a72063dbcc9e72792847a8852c)
- (5093d32) [Refactor](https://github.com/envato/jwt_signed_request/commit/5093d323926c79e9870e873a92876e48b2cc89dc)
- (090d909) [Test key store lookup](https://github.com/envato/jwt_signed_request/commit/090d9097147b150561300ce6b4f29ba978ae7fb1)
- (891627a) [Support custom key store when verifying](https://github.com/envato/jwt_signed_request/commit/891627a8b27570b3dd71840d3a93dbeaa09caac8)
- (dd1ee14) [Typo](https://github.com/envato/jwt_signed_request/commit/dd1ee141eb851abc843ddb3f87f3c7d9aede7cc8)
- (96fae18) [Remove redundant freeze](https://github.com/envato/jwt_signed_request/commit/96fae183ffa195eea16782b3c6a10e39d3ea3354)
- (9d0ad0c) [Formatting](https://github.com/envato/jwt_signed_request/commit/9d0ad0c16fda955be06793d442ff2aad7e88ea52)
- (f3d10d4) [Formatting](https://github.com/envato/jwt_signed_request/commit/f3d10d4aec6b5f2f35373b71be207d8ab8550e2f)


</details>